genesyscloud/user/resource_genesyscloud_user_utils.go

```diff
@@ -236,6 +241,32 @@ func updateUserProfileSkills(d *schema.ResourceData, proxy *userProxy) diag.Diag
 	return nil
 }
 
+func updateUserVoicemailPolicies(d *schema.ResourceData, proxy *userProxy) diag.Diagnostics {
+	if d.HasChange("voicemail_userpolicies") {
+		if voicemailUserpolicies := d.Get("voicemail_userpolicies").([]interface{}); voicemailUserpolicies != nil {
```

Comment:
This is a really deep method with multiple levels of if.

Go developer usually do a straight line where it does an if and then returns and then breaks.  Multiple-level if can be problematic and make the code harder to read